content/applications/inventory_and_mrp/purchase/products/pricelist.rst

```diff
@@ -0,0 +1,260 @@
+right of the field.
+
+.. note::
+   To update to existing records, tick the :guilabel:`I want to update data (import-compatible export)` checkbox, and refer to
+   the section on the :ref:`External ID <purchase/products/external-id>` field.
+
+   For details on commonly-used fields for importing vendor pricelists, see the :ref:`Common fields
+   <purchase/products/common-fields>` section.
+
+Select the desired :guilabel:`Export Format`: :guilabel:`XLSX` or :guilabel:`CSV`.
+
+To save the selected fields as a template, click the :guilabel:`Template` field, and select
+:guilabel:`New template` from the drop-down menu. Type the name of the new template, and click the
+:icon:`fa-floppy-o` :guilabel:`(save)` icon. After that, the template is a selectable option
+when clicking the :guilabel:`Template` field.
+
+Finally, click :guilabel:`Export`.
+
+.. note::
+   With :ref:`developer mode <developer-mode>` turned on, the column names of the exported file
+   display the *Technical name of the field*, not the *Field name*.
+
+.. example::
+   .. figure:: pricelist/export-data.png
+      :align: center
+      :alt: Exporting vendor pricelist.
+
+      Export vendor pricelist in XLSX format. It includes **Product Template** and other fields
+      in the "Fields to export" section.
+
+.. _purchase/products/external-id:
+
+External ID
+~~~~~~~~~~~
+
+*External ID* is a unique identifier used to update existing vendor pricelists. Without
+it, imported records create new entries, instead of updating existing ones. Including this field
+in the XLSX or CSV, indicates the line replaces an existing vendor pricelist in the Odoo database.
+
+.. example::
+   .. figure:: pricelist/duplicate-values.png
+      :align: center
+      :alt: Show 'Ready Mat' appear twice.
+
+      'Ready Mat' appears twice because the External ID was omitted during the price update from
+      `$790` to `$780`.
+
+To look-up the :guilabel:`External ID` for a vendor pricelist, tick the :guilabel:`I want to update
+data (import-compatible export)` checkbox at the top of the :guilabel:`Export Data` pop-up window.
+
+.. note::
+   Selecting :guilabel:`External ID` from the :guilabel:`Available fields` section with the
+   :guilabel:`I want to update data (import-compatible export)` checkbox ticked results in an export file with two columns
+   containing the external ID.
+
+.. _purchase/products/common-fields:
+
+Common fields
+~~~~~~~~~~~~~
+
+Below is a list of commonly-used fields when importing vendor pricelists:
+
+.. list-table:: Field name definitions
+   :header-rows: 1
+
+   * - Field name
+     - Used for
+     - Field in Odoo database
+     - Technical name of field
+   * - Vendor
+     - The only required field for creating a vendor pricelist record. This field specifies the
+       vendor associated with the product.
+     - :guilabel:`Vendor` field in the :ref:`vendor pricelist of the product form
+       <purchase/products/pricelist>`.
+     - partner_id
```

Comment:
Traditionally the technical name is formatted with literals throughout our documentation :)
```suggestion
     - `partner_id`
```